### DISCUSSION TITLE
This user account is blocked.

### DISCUSSION THREAD

**Discussion by bombel1337** [vercel/community#3794] [Account Issue] [answered]
### Question

Why i can't register on vercel app? It says This user account is blocked.
It would be nice to get any clarifications.

![image](https://github.com/vercel/community/assets/115701807/8d35edaa-fe5d-4ed3-9af6-1881b3edc6b6)

**Comment by amyegan** [comment, marked as answer]
Due to the sensitive nature of account information, we cannot safely discuss it in this public forum. For more info, please check this guide: [Why has my account or deployment been paused?](https://vercel.com/guides/why-is-my-account-deployment-blocked)
